b2b-sample/guardio-insurance-sample-application/README.md

```diff
@@ -17,3 +17,9 @@ yarn install
 yarn dev
 ```
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+
+## Additionally
+* If the correct organisation id and organisation name are returned in the idToken when a user is logging into an organisation, remove the `SAMPLE ORGS` section from the <i>config.json</i> file.
+
+#### If you are using the app with [Asgardeo](https://wso2.com/asgardeo/)
+* Please change the value of the <i>config.json</i>  file's `APP IN ASGARDEO FIRST LEVEL` to `true` if the organisation that contains the application is in the first level.
```

Comment:
`organization`

Reply:
These have been fixed with the new README.md file

### File
- https://github.com/wso2/samples-is/tree/master/b2b-sample#readme